pallets/chain-space/src/tests.rs

```diff
@@ -1082,3 +1082,48 @@ fn restoring_approval_of_a_space_by_non_root_should_fail() {
 		);
 	});
 }
+
+#[test]
+fn add_delegate_should_fail_if_space_delegates_limit_exceeded() {
+	let creator = DID_00;
+	let author = ACCOUNT_00;
+	let space = [2u8; 256].to_vec();
+	let space_digest = <Test as frame_system::Config>::Hashing::hash(&space.encode()[..]);
+
+	let id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&space_digest.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+
+	let space_id: SpaceIdOf = generate_space_id::<Test>(&id_digest);
+
+	new_test_ext().execute_with(|| {
+		// Create the space
+		assert_ok!(Space::create(
```

Comment:
Even though space::create() has check for `spaceDelegatesLimitExceeded`, It is not possible hit that case since creation of every chain space is unique, based on creator & space_id.

Reply:
Thanks for the info, makes great sense tbh. Thanks!